### ISSUE TITLE
Error in nf-core/hic MultiQC: invalid literal for int() 

### ISSUE THREAD

**Issue by lauracarpen** [ewels/MultiQC#1922]
### Description of bug

We are using nh-core/hic, which uses MultiQC. Our pipeline stops with the error below. The pipeline seems to be using v1.13, which is not the latest.

### File that triggers the error

[untreated_allValidPairs.mergestat.zip](https://github.com/ewels/MultiQC/files/11441955/untreated_allValidPairs.mergestat.zip)

### MultiQC Error log

```console
/// MultiQC 🔍 | v1.13

|           multiqc | MultiQC Version v1.14 now available!
|           multiqc | Search path : /path/to/hic_test/work/8e/1c2f5aef67740224fd4ad9874b9b02
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 21/21  
|    custom_content | nf-core-hic-summary: Found 1 sample (html)
╭───────────────── Oops! The 'hicpro' MultiQC module broke... ─────────────────╮
│ Please copy this log and report it at                                        │
│ https://github.com/ewels/MultiQC/issues                                      │
│ Please attach a file that triggers the error. The last file found was:       │
│ ./input_2/untreated/untreated_allValidPairs.mergestat                        │
│                                                                              │
│ Traceback (most recent call last):                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/multiqc.py", line 65 │
│     output = mod()                                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/hicpro/hicpr │
│     self.parse_hicpro_stats(f, k)                                            │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/hicpro/hicpr │
│     self.hicpro_data[s_name][s[0]] = int(s[1])                               │
│ ValueError: invalid literal for int() with base 10: '2.47438e+09'            │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
|            fastqc | Found 2 reports
|           multiqc | Compressing plot data
|           multiqc | Report      : multiqc_report.html
|           multiqc | Data        : multiqc_data
|           multiqc | Plots       : multiqc_plots
|           multiqc | MultiQC complete
```

### Before submitting

- [X] I have read the [troubleshooting documentation](https://multiqc.info/docs/#troubleshooting-1).
- [X] I am using the latest release of MultiQC.
- [X] I have included a full MultiQC log, not truncated.
- [X] I have attached an input file (**.zip** if necessary) that triggers the error.

**Comment by ewels** [comment]
Thank you!

**Comment by Just-Roma** [comment]
Hi @lauracarpen, [here](https://github.com/Just-Roma/MultiQC/tree/Fix-issue-1922) is a fix for the bug.

I have tested your data on MultiQC v1.14, and the same error has popped up.
I am not familiar with this module, but have tested new code with the [test-data](https://github.com/Just-Roma/MultiQC_TestData/tree/master/data/modules/hicpro). The output is the same as from the master branch. The html-output for your data is partially broken, the reason for it is probably some violation of Highcharts API, but I did not investigate the problem. Though when combined with the mentioned test-data, the html looks fine.
